flit_core/flit_core/config.py

```diff
@@ -5,9 +5,9 @@
 import os
 import os.path as osp
 from pathlib import Path
-import tomli
 import re
 
+from .vendor import tomli
```

Comment:
Do you want to bother with underscore prefixes?

Reply:
```suggestion
try:
    from .vendor import tomli
except ImportError:
    import tomli
```

How about this? It would make unbundling a little bit easier.

You could even do it the opposite way — i.e. prevent the "newest" tomli available on the system, and fall back to vendoring only when really necessary.

Reply:
Sorry, I have no clue what distracted me at the time. I meant "prefer".

Reply:
If someone is debundling (i.e. modifying the package), they can also do the extra work of updating references. There’s no reason to bake support in unless someone with that use case comes and asks for it; and provides a clear justification.

Reply:
I'm inclined to leave it as is, because catching the error and falling back could also mask a real error (e.g. if tomli gained a dependency and I bundled it without noticing). Unlikely, but as @pradyunsg says, it's also not much work for someone who wants to unbundle it.